### DISCUSSION TITLE
Upgraded database and now I'm getting errors when trying to log in

### DISCUSSION THREAD

**Discussion by MaseCodes20** [supabase/supabase#19135] [Questions]
![1 aaError](https://github.com/supabase/supabase/assets/95643884/9331491c-9573-437a-99e6-059927844728)

![sessions model](https://github.com/supabase/supabase/assets/95643884/6ae1ad30-bc72-4fb4-a584-e673763f39cf)

**Comment by GaryAustin1** [comment]
Try restarting your database.  I think that is a required step.

**Reply by MaseCodes20** [reply]
I did that and I'm still getting the error, I also see in the error log that my session table is now missing a couple columns. 

refreshed_at, ip and user_agent.

![1aaError2](https://github.com/supabase/supabase/assets/95643884/a583d6b6-3529-4533-94d2-493f76a064d5)

**Reply by GaryAustin1** [reply]
I'd contact support at this point.  They were having a problem with pause/restore in US-EAST about the time of your issue.

**Comment by kualta** [comment]
same exact problem, yet I haven't modified the schema before it started appearing

**Reply by kualta** [reply]
resolved by adding the columns back manually in my prisma schema:
```prisma
model sessions {
  refreshed_at   DateTime?        @db.Timestamptz(6)
  user_agent     String?          @db.VarChar(255)
  ip             String           @db.Inet()
  <...>
}
```
but again, I'm not sure why this happened and I'm not sure when this is going to break again. This probably should be an issue.
